CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -16,7 +16,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 - Fix [url redirects while checking peer urls during new member addition](https://github.com/etcd-io/etcd/pull/16986)
 
 ### Dependencies
-- Compile binaries using [go 1.20.12](https://github.com/etcd-io/etcd/pull/17077)
+- Compile binaries using [go 1.20.13](https://github.com/etcd-io/etcd/pull/17275)
```

Comment:
`3.5.11` has already been released so we need to leave this line as is and make a new entry for `### Dependencies` under `3.5.12`.

Reply:
Don't think we need separate lines each dependency update, at least historically we only kept the latest update of one kind.